metricflow/test/model/test_data_source_container.py

```diff
@@ -187,3 +187,11 @@ def test_local_linked_elements_for_metric(new_metric_semantics: MetricSemantics)
         "listing__ds",
         "listing__capacity_latest",
     }
+
+
+def test_get_data_sources_for_identifier(new_data_source_semantics: DataSourceSemantics) -> None:  # noqa: D
+    identifier_reference = IdentifierReference(element_name="user")
+    linked_data_sources = new_data_source_semantics.get_data_sources_for_identifier(
+        identifier_reference=identifier_reference
+    )
+    assert len(linked_data_sources) == 10
```

Comment:
There are 9 distinct data sources with a user entity pointer - bookings has the `user` identity twice, and as such `bookings` will appear twice in the result set.

Note - as far as our current join resolution is concerned, `bookings` cannot be joined to `user`. #240 will fix this, though, so I think we should keep it as-is unless that discrepancy is going to be a significant problem for you.

Reply:
I don't believe it's a significant problem (at this time).